api/errors/errors.go

```diff
@@ -1125,168 +1070,21 @@ var errorCodes = errorCodeMap{
 		Description:    "The provided 'x-amz-content-sha256' header does not match what was computed.",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
-
-	// MinIO extensions.
-	ErrStorageFull: {
-		ErrCode:        ErrStorageFull,
-		Code:           "XMinioStorageFull",
-		Description:    "Storage backend has reached its minimum free disk threshold. Please delete a few objects to proceed.",
-		HTTPStatusCode: http.StatusInsufficientStorage,
-	},
-	ErrParentIsObject: {
-		ErrCode:        ErrParentIsObject,
-		Code:           "XMinioParentIsObject",
-		Description:    "Object-prefix is already an object, please choose a different object-prefix name.",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
-	ErrRequestBodyParse: {
-		ErrCode:        ErrRequestBodyParse,
-		Code:           "XMinioRequestBodyParse",
-		Description:    "The request body failed to parse.",
-		HTTPStatusCode: http.StatusBadRequest,
-	},
```

Comment:
I decided to remove the error because now we parse requests bodies as `xml` and we already have `MalformedXML` error and I'm not sure if we'll need this error further or not.